magma/array.py

```diff
@@ -3,19 +3,19 @@
 import operator
 from abc import ABCMeta
 from hwtypes import BitVector
-from .common import deprecated
-from .ref import AnonRef, ArrayRef
-from .t import Type, Kind, Direction, In, Out
+from .ref import ArrayRef
+from .t import Type, Kind, Direction
 from .compatibility import IntegerTypes
 from .digital import Digital
 from .bit import Bit
-from .bitutils import int2seq, seq2int
+from .bitutils import int2seq
 from .debug import debug_wire, get_callee_frame_info
 from .logging import root_logger
 from .protocol_type import magma_type, magma_value
 
 from magma.operator_utils import output_only
-from magma.wire_container import WiringLog, Wire, Wireable
+from magma.wire_container import (WiringLog, AggregateWireable,
+                                  aggregate_wireable_method)
```

Comment:
nit:

```suggestion
from magma.wire_container import (
    WiringLog,
    AggregateWireable,
    aggregate_wireable_method,
)
```

and should go after `magma.protocol_type` import